tslib/src/functions/check-started/check-started.service.ts

```diff
@@ -26,7 +26,8 @@ export class CheckStartedService {
       clientCheckStartedAt: checkStartedMessage.clientCheckStartedAt
     })
     const preparedCheck = await this.redisService.get(preparedCheckKey)
-    if (preparedCheck.config.practice === false) {
+
+    if (preparedCheck && preparedCheck.config.practice === false) {
```

Comment:
I think this should either be written in pure js style:

`if (preparedCheck && preparedCheck.config && preparedCheck.config.practice === false) {`

Or, something like

`if(R.path(['config', 'practice'], preparedCheck) === true`

Reply:
The ramda approach is 👌🏼